tox.ini

```diff
@@ -17,13 +17,12 @@ python =
 [testenv]
 whitelist_externals = git
 # In case requirements.txt got updated, one wants to recreate (it was not automatical)
-recreate = True 
+recreate = True
 deps = -rrequirements.txt
 commands =
 	git submodule sync -q
```

Comment:
There are no submodules in theboss now, so git commands can also be removed. Would you mind doing it?

Reply:
done